### PR TITLE
Creating a Location resource

### PR DESCRIPTION
**Pull request by Wittev1**
1. migration
2. form
3. routes
4. store

### CODE REVIEW

File: resources/views/locations/index.blade.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        <div class="col-md-8">
+            <div class="card">
+                <div class="card-header">Locaties</div>
+
+                <div class="card-body">
+                	<div class="container">
+                	    @foreach ($locations as $location)
+                	       <a class="btn" href="{{ route('locations.edit', $location->id) }}"><i class="far fa-edit"></i></a>
+                	       <form action="{{ route('locations.destroy', $location->id) }}" method="POST"
+        	                     style="display: inline"
```
Comment (wotta) [inline resources/views/locations/index.blade.php]: Why the inline style?

File: resources/views/locations/index.blade.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        	                     onsubmit="return confirm('Zeker weten?');">
+        	                   <input type="hidden" name="_method" value="DELETE">
```
Comment (wotta) [inline resources/views/locations/index.blade.php]: `@method('delete')`

File: resources/views/locations/index.blade.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+        	                   {{ csrf_field() }}
```
Comment (wotta) [inline resources/views/locations/index.blade.php]: `@csrf`

File: resources/views/locations/index.blade.php [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+<div class="container">
+    <div class="row justify-content-center">
+        <div class="col-md-8">
+            <div class="card">
+                <div class="card-header">Locaties</div>
+
+                <div class="card-body">
+                	<div class="container">
+                	    @foreach ($locations as $location)
+                	       <a class="btn" href="{{ route('locations.edit', $location->id) }}"><i class="far fa-edit"></i></a>
```
Comment (wotta) [inline resources/views/locations/index.blade.php]: `{{ route('locations.edit', $location) }}`

File: resources/views/locations/index.blade.php [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+                	       <form action="{{ route('locations.destroy', $location->id) }}" method="POST"
```
Comment (wotta) [inline resources/views/locations/index.blade.php]: same as the comment above.

File: resources/views/locations/index.blade.php [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+                <div class="card-body">
+                    <div class="container">
+                        @if (session('message'))
+                            <div class="alert alert-success" role="alert">
+                                {{ session('message') }}
+                            </div>
+                        @endif
+
+                        @if (count($locations))
```
[the remark this answers is not in the document]
Comment (Wittev1) [reply resources/views/locations/index.blade.php]: Cool, thanks

File: app/Http/Controllers/LocationController.php [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+    }
+
+    /**
+     * Update the specified resource in storage.
+     */
+    public function update(LocationUpdateRequest $request, Location $location) : RedirectResponse
+    {
+        $location->update($request->validated());
+
+        return redirect()->route('locations.index')->withMessage('Locatie bijgewerkt'); // Definitly not "geüpdatet"   ( ͡° ͜ʖ ͡°)
```
[the remark this answers is not in the document]
Comment (Wittev1) [reply app/Http/Controllers/LocationController.php]: Ok ok, removed the comment :P

**Pull request merged**